src/scripts/angry-tock.test.js

```diff
@@ -217,7 +217,7 @@ describe("Angry Tock", () => {
     });
   });
 
-  it("sends a calm/disappointed message first, then an angry message and truant report", async () => {
```

Comment:
question: Does it still send an ops report? Or was the comment removed because that's not part of the behavior here?

Reply:
The latter. The ops reporting was moved out of "angry Tock" and into a separate script. I didn't do a good job of scrubbing the tests for clarity!